e2e/specs/app.e2e-common.ts

```diff
@@ -12,278 +13,278 @@ describe('FogLAMP gui', () => {
+    //     'HTTP_SOUTH Device',
+    //     'OCS North Plugin Configuration',
+    //     'South Plugin polling template',
+    //     'COAP Device',
+    //     'TI SensorTag CC2650 polling South Plugin',
+    //     'TI SensorTag CC2650 async South Plugin',
+    //     'Scheduler configuration',
+    //     'Service Monitor configuration'
+    //   ];
+    //   skipLogin.navigateToConfig();
+    //   for (const ConfigTitle in ConfigTitles) {
+    //     expect(skipLogin.getConfigTitles()).toContain(ConfigTitles[ConfigTitle]);
+    //   }
+    //   expect(skipLogin.isAddButtonPresent()).toEqual(true);
+    //   expect(skipLogin.isSaveButtonPresent()).toEqual(true);
+    //   expect(skipLogin.isCancelButtonPresent()).toEqual(true);
+
+    //   expect(skipLogin.editAndVerifyConfigValue()).toEqual('Value updated successfully');
+
+    //   skipLogin.clickAddButton();
+    //   expect(skipLogin.addConfigInputTagCount()).toEqual(3);
+    //   expect(skipLogin.addConfigSelectTagCount()).toEqual(1);
+    //   expect(skipLogin.addConfigTextareaCount()).toEqual(1);
+    //   expect(skipLogin.isAddConfigSaveButton()).toEqual(true);
+    // });
+
+    it('Should Display Scheduled Tasks', () => {
+      skipLogin.navToScheduledTasks();
+      expect(skipLogin.getSchedulesTitle()).toContain('Schedules');
+      expect(skipLogin.getSchedulesRefreshButton()).toEqual(true);
+      expect(skipLogin.getCreateScheduleButton()).toContain('Create');
+      expect(skipLogin.getTasksTitle()).toContain('Tasks');
+      expect(skipLogin.getTasksRefreshButton()).toEqual(true);
+      expect(skipLogin.getTasksSelectTag()).toEqual(1);
+
+      expect(skipLogin.createAndVerifySchedule()).toEqual('Schedule created successfully.');
+      skipLogin.closeAlert();
+      expect(skipLogin.updateAndVerifySchedule()).toEqual('Schedule updated successfully.');
+      skipLogin.closeAlert();
+      expect(skipLogin.isUpdatedSchedulePresent()).toContain('updateSchedule');
+      expect(skipLogin.disableAndVerifySchedule()).toEqual('Schedule successfully disabled');
+      skipLogin.closeAlert();
+      expect(skipLogin.deleteAndVerifySchedule()).toEqual('Schedule deleted successfully.');
+    });
+
+    it('Should Display Service Health', () => {
+      const ColumnsName = [
+        'Name',
+        'Status',
+        'Type',
+        'Protocol',
+        'Address',
+        'Service Port',
+        'Management Port'
+      ];
+      skipLogin.navToServiceHealth();
+      expect(skipLogin.getServiceStatusTitle()).toContain('Services Status');
+      expect(skipLogin.getAddServiceButton()).toContain('Add Service');
+      expect(skipLogin.getServiceStatusRefreshButton()).toEqual(true);
+
+      for (const ColumnName in ColumnsName) {
+        expect(skipLogin.getServiceHealthColNames()).toContain(ColumnsName[ColumnName]);
+      }
+      expect(skipLogin.httpSouthServiceStatus()).toContain('running');
+
+      //   Failing due to address binding.
```

Comment:
This is not failing due to address binding but because of no south service plugin configured by default in the system now. So either we need to configured the httpSouth plugin or else we need to remove this from here.

cc/ @praveen-garg

Reply:
remove! we will add fixtures in another ticket for services, logs, readings and all such data ..